spec/controllers/catalog_controller_spec.rb

```diff
@@ -45,7 +45,8 @@
          'human_readable_language',
          'generic_type',
          'member_of_collections',
-         'repository']
+         'repository',
+         'program_tesim']
```

Comment:
update this to only program, also need to add program to blacklight en yml files

Reply:
in the blacklight.en.yml program_sim is already there:
does it need to be added elsewhere?
```
    search:
      facets:
        title: 'Browse items'
      clear_all: 'Clear all'
      zero_results:
        title: 'No results found.'
        modify_search: ''
        use_fewer_keywords: 'Remove some filters or terms from your query and search again.'
        search_fields: 'you searched by %{search_fields}'
        search_everything: 'try searching everything'
      form:
        search:
          placeholder: 'Search'
          placeholder_collection: 'Search this collection'
      new_search: 'Start New Search'
      fields:
        facet:
          (...others before this)
          program_sim: 'Program'
          program_tesim: 'Program'
```